### ISSUE TITLE
AlexaPi + Pocketsphinx Error (ImportError: cannot import name get_model_path)

### ISSUE THREAD

**Issue by likeadoc** [alexa-pi/AlexaPi#165]
Latest Raspbian Jessie on RPi3 + PulseAudio
Python 2.7.9
Pocketsphinx 5prealpha

**Description of problem:**
```
PocketSphinx on its own works without problems - but AlexaPi produces an Import Error
```

**Traceback (if applicable):**
```
Traceback (most recent call last):
Feb 02 05:41:48 magicmirrorpie python[610]: File "/opt/AlexaPi/src/main.py", line 23, in <module>
Feb 02 05:41:48 magicmirrorpie python[610]: from pocketsphinx import get_model_path
Feb 02 05:41:48 magicmirrorpie python[610]: ImportError: cannot import name get_model_path

```

**Comment by lion6230i** [comment]
You can use python command line try to manual import pocketsphinx first.

**Comment by likeadoc** [comment]
I tried to manually (via pdb) execute  " import pocketsphinx"  and still get the same message ( ImportError: cannot import name get_model_path)

**Comment by renekliment** [comment]
@likeadoc I think you might have an old version. Be sure to have https://pypi.python.org/pypi/pocketsphinx/0.1.3 installed.

**Comment by likeadoc** [comment]
pocketsphinx version is 0.1.3 ( installed  with  AlexaPi setup.sh) - If i change get_model_name to a variable containing the path to the models i dont get an error anymore.

**Comment by likeadoc** [comment]
But you are right. it seems to be a problem with pocketsphinx because the test snippets return the same error

**Comment by likeadoc** [comment]
solved ! was a faulty pocketsphinx 0.1.3 install! after uninstall and install via pip it works like a charm!
